docs/settings.rst

```diff
@@ -0,0 +1,142 @@
+.. _settings:
+
+--------
+Settings
+--------
+
+Wafer has several Django settings that control its behaviour.
+It attempts to provide reasonable defaults for these, (and Django in general),
+in the ``wafer.settings`` module, so you can import this in your app's
+``settings.py``, and then override things you want to change.
+
+
+Wafer's settings
+================
+
+``PAGE_DIR``
+    The directory that the ``load_pages`` management command will load
+    pages from.
+    Should be an absolute path with a trailing ``/``.
+
+``TALKS_OPEN``
+    Controls whether talk submissions are accepted.
+    Set to ``False`` to close talk submissions.
+
+``WAFER_CACHE``
+    The name of the Django cache backend that wafer can use.
+    Defaults to ``'wafer_cache'``.
+
+``WAFER_DEBIAN_NM_API_KEY``
+    The API Key for ``nm.debian.org``'s API, used by Debian SSO.
+    Used when ``WAFER_SSO`` contains ``'debian'``.
+
+``WAFER_DYNAMIC_MENUS``
+    A list of functions to call to generate additional menus.
+
+``WAFER_GITHUB_CLIENT_ID``
+    The client ID for GitHub SSO.
+    Used when ``WAFER_SSO`` contains ``'github'``.
+    If you set this up, they'll provide you with one.
+
+``WAFER_GITHUB_CLIENT_SECRET``
+    The secret for GitHub SSO.
+    Used when ``WAFER_SSO`` contains ``'github'``.
+    If you set this up, they'll provide you with one.
+
+``WAFER_HIDE_LOGIN``
+    A boolean flag.
+    When ``True``, the login link in the menu is hidden.
+    This is useful to set, before making a site static.
+
+``WAFER_SSO``
+    A list of SSO mechanisms in use.
+    Possible options are: ``'debian'``, ``'github'``.
+
+``WAFER_TALK_FORM``
+    The form used for talk/event submission.
+    There is a reasonable default form, but this can be changed to
+    customise the submission process.
+
+``WAFER_MENUS``
+    Static menu structure for the site.
+    This is a list of dicts, with the keys:
+
+    ``label``
+        The text in the menu.
+
+    ``url``
+        The URL to link to.
+
+    ``items``
+        An optional list of similar dicts, making up a submenu.
+
+``WAFER_PUBLIC_ATTENDEE_LIST``
+    A boolean flag.
+    When ``True``, all registered users' profiles are publicly visible.
+    Otherwise, only users with associated talks have public profiles.
+
```

Comment:
"associated public talks", perhaps - we don't expose users until we've accepted their talk.